### ISSUE TITLE
All text and icons got larger when updating from 2019.910 to 2019.191

### ISSUE THREAD

**Issue by memesnightmares** [ppy/osu#6241]
**Describe the bug:** 

When I updated osu!lazer, everything suddenly seemed bigger. Specifically, the "GOOD" or "GREAT" over the circles (in taiko mode), but also everywhere else in osu!lazer.

**Screenshots or videos showing encountered issue:** 

**osu!lazer version:** 2019.919/2019.910

**Logs:**

**Comment by peppy** [comment]
The change in taiko was intentional for now. As for the rest of your game, did you change your UI scale?

**Comment by memesnightmares** [comment]
No, although I could just be seeing things -- I only really noticed in taiko. If it's intentional, I guess I just don't like it, but thats OK :)
